src/toast/ops/madam.py

```diff
@@ -899,7 +899,7 @@ def _stage_data(
                     for psdstart, psd, detw in psdlist:
                         psdstarts.append(psdstart)
                         psdvals.append(psd)
-                    detweights[idet] = psdlist[0][2]
+                    detweights[idet] = psdlist[0][2].to_value(1 / u.K**2)
```

Comment:
Thanks, this was fixed in the other PR as well.  Better to get fixed twice than not at all :-)

Reply:
It was preventing me from running the unit tests